polyfill/test/validStrings.mjs

```diff
@@ -408,7 +417,17 @@ const duration = seq(
 );
 
 const instant = seq(date, dateTimeSeparator, timeSpec, dateTimeUTCOffset, [timeZoneAnnotation], [annotations]);
+const instantOffsetMinutePrecision = seq(
+  date,
+  dateTimeSeparator,
+  timeSpec,
+  timeZoneUTCOffsetName,
+  [timeZoneAnnotation],
+  [annotations]
+);
 const zonedDateTime = seq(dateTime, timeZoneAnnotation, [annotations]);
+const dateTimeOffsetMinutePrecision = seq(date, [dateTimeSeparator, timeSpec, [timeZoneUTCOffsetName]]);
+const zonedDateTimeOffsetMinutePrecision = seq(dateTimeOffsetMinutePrecision, timeZoneAnnotation, [annotations]);
```

Comment:
If possible, I'd prefer to keep this section as close to the ISO 8601 grammar in the spec as possible. Can we combine these new items back into `instant` and `zonedDateTime`?